src/ui/public/kfetch/kfetch_error.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export class KFetchError extends Error {
```

Comment:
Thanks for renaming this! Can we update references to `FetchError` to `KFetchError` in https://github.com/elastic/kibana/blob/master/src/ui/public/error_auto_create_index/error_auto_create_index.test.js#L33 ? They're only semantic references, so the code isn't broken or anything.

Reply:
Done. I also made a minor change to the test. Is that ok?

Reply:
👏 Thanks for doing that!